app/views/user/subscriptions/index.html.erb

```diff
@@ -42,16 +42,20 @@
                   <%= t('.whole_site') %>
                 <% end %>
               </div>
-              <% if @user_notification_modules.length > 1 %>
-                <%= f.collection_check_boxes(:modules, @user_notification_modules, :first, :first) do |b| %>
-                  <div class="option ">
-                    <%= b.check_box({"data-selected" => f.object.modules.include?(b.value)}) %>
-                    <%= b.label do %>
-                      <span></span>
-                      <%= t(".modules.#{b.text}") %>
-                    <% end %>
-                  </div>
-                <% end %>
+              <% if @user_notification_modules.any? %>
```

Comment:
Maybe in the future we could divide this in partials and declare the partials in each module, as we do with the partials that implement the navigation.